website/assets/scss/_styles_project.scss

```diff
@@ -50,3 +50,34 @@
         font-size: 1.2rem;
     }
 }
+
+//
+// Adopters logos
+//
+
+.adopters-logos {
+    // Horizontally center logos.
+    text-align: center;
+}
+
+.adopter-logo {
+    display: inline-block;
+    margin: 10px;
+    padding: 10px;
+    width: 180px;
+    height: 100px;
+    border: 1px solid #dedede;
```

Comment:
On your screenshot the border looks like leftover from debugging. Do you plan to keep it?

Reply:
Yes, I wanted to introduce visual boxes. WDYT?

Reply:
As it is now, they are too subtle, and look like forgotten leftover. If you want to keep them, I would suggest using higher contrast. (Personally I would leave them out, I don't think boxes around these logos add anything).

Reply:
The box visually helps to give each logo the same weight. I've added a light drop shadow to better emphasis it.

![Screenshot 2021-01-21 at 12 53 46](https://user-images.githubusercontent.com/1701904/105347923-dee42a00-5be7-11eb-83ae-76759d6f31ab.png)

Reply:
That's subtle, but looks better to me. Now it doesn't look like a bug anymore :)